detection-rules/attachment_office_remote_html_template.yml

```diff
@@ -33,7 +33,7 @@ source: |
               // detect files without an extension
               or .file_type in ("docx")
           )
-          and any(file.oletools(.).relationships, regex.icontains(.target, "html?!"))
+          and any(file.oletools(.).relationships, regex.icontains(.target, "html?!", "doc", "docm", "docx", "dot", "dotm", "pptm", "ppsm", "xlm", "xls", "xlsb", "xlsm", "xlt", "xltm"))
```

Comment:
what do `htm!` and `html!` mean? looks like the `!` is interpreted literally in RE2, not sure if that's intended.

Reply:
maybe @jkamdjou knows since those were in the original rule

Reply:
yeah not sure...